guides/searcher-onboarding.md

```diff
@@ -177,16 +230,16 @@ The signature needs to be provided via the 'X-Flashbots-Signature' Header. Refer
 ### Why is there rate limiting?
 Rate limiting is currently in place to protect the infrastructure sitting behind it. Similar to how Infura has a gas limit on eth_call.
 
-
 ### Why do I need to provide this unrelated auth-signing key?
-
 By signing payloads with your own relay signing key, this will enable building a reputation for high-priority delivery of your bundles to miners. MEV-relay simulates bundles before sending to miners which can take a small amount of time. MEV-relay cannot determine which bundles are profitable without performing a full simulation. This signing key allows the relay to infer which bundles are likely profitable, based on historical performance. Using a reputation system allows reliable searchers to be rewarded while still allowing new searchers to participate.
 
+### Are you on any testnets?
+Yes, we are on Goerli. Our endpoint is `https://relay-goerli.flashbots.net/`.
+
 ### How should I know the correct RPC that accepts eth bundle commands?
-You have to set it to a regular Ethereum node.
+You have to set it to a regular Ethereum node. You do not need to run MEV-Geth as a searcher.
```

Comment:
What is `eth bundle commands`? I think this question/answer is misleading. I appreciate the "you don't need to run mev-geth as searcher", because it comes up all the time.

Reply:
"The relay service only accepts `eth_sendBundle` and `eth_callBundle` calls and will reject all other requests, such as `eth_getTransactionCount` or `eth_getBlockByNumber`. You will need a standard Ethereum endpoint to service any of of these "basic" Ethereum requests your searcher needs to perform its work. After this generic work is completed, the bundle is sent to the relay at `https://relay.flashbots.net/`.

Initializing any of the Flashbots libraries requires you to provide a standard Ethereum endpoint, to service basic requests required to form the bundle, as the relay will not `eth_getTransactionCount`, which is required to sign Ethereum transactions.